docs/deployment/config_descriptions.md

```diff
@@ -261,23 +322,29 @@ This snippet is an example of configuration for a list of SNS Subscriptions. We
 
 The main difference between this and the previous example is the inclusion of the `sns.arn` attribute - this tells our deployment/compiling step that we're configuring subscriptions, not a new topic. More information for each of the individual attributes can be found in [AWS SNS Subscription Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sns-subscription.html).
 
-## buckets
+### iams
 
-The config buckets should map to the same names you used when creating buckets in the [Prepare AWS](#prepare-aws-configuration) step. Buckets are defined in the config.yml with a key, name, and type. Types should be one of: internal, public, private, or protected. Multiple buckets of each type can be configured.
+Optional. Overrides allowed if your IAM role ARNs do not match the following convention used in `@cumulus/deployment/app/config.yml`:
 
-## iams
+```yaml
```

Comment:
good call. I had a comment somewhere in another file suggesting that we add just this :+1: